src/coredump/coredumpctl.c

```diff
@@ -197,6 +201,22 @@ static int parse_argv(int argc, char *argv[]) {
 
                         break;
 
+                case 'S':
+                        r = parse_timestamp(optarg, &arg_since);
+                        if (r < 0) {
+                                log_error("Failed to parse timestamp: %s", optarg);
+                                return -EINVAL;
+                        }
```

Comment:
you could shorten this to this btw:

```
r = parse_timestamp(optarg, &arg_since);
if (r < 0)
        return log_error_errno(r, "Failed … );
```

That's not only shorter, but also propagates the error number to the logs and the parent...